packages/runtime-dom/src/patchProp.ts

```diff
@@ -4,12 +4,24 @@ import { patchAttr } from './modules/attrs'
 import { patchDOMProp } from './modules/props'
 import { patchEvent } from './modules/events'
 import { isOn, isString, isFunction, isModelListener } from '@vue/shared'
-import { RendererOptions } from '@vue/runtime-core'
+import { RendererOptions, VNodeTypes } from '@vue/runtime-core'
 
 const nativeOnRE = /^on[a-z]/
 
 type DOMRendererOptions = RendererOptions<Node, Element>
 
+const IMGPRIORPROPS = ['loading']
+export function getPriorProps(type: VNodeTypes) {
```

Comment:
I think "priority props" could be a better name. It's sometimes confusing to see a `prior` prop when we are discussing the patching process, because it usually means the props set in the previous pass.